pages/pipelines/command_step.md.erb

```diff
@@ -2,26 +2,113 @@
+
+A *command* step runs a command on an agent. The command can be the name of a script, or <cli things>
+
+A basic command step can be defined in your pipeline settings, or in your [pipeline.yml](/docs/pipelines/uploading-pipelines) file.
+
+<%= image "command-step.png", width: 944/2, height: 364/2, alt: "Screenshot of a basic command step" %>
 
 ```yml
-- command: "command.sh"
+steps:
+  - command: "command.sh"
 ```
 
-The following step includes every configuration property for *command* steps:
+
+## Command Step Attributes
+
+_Optional attributes:_
+
+<table>
+  <tr>
+    <td><code>name</code></td>
+    <td>
+      The text and/or emoji that will be displayed in the pipeline visualisation in Buildkite.<br>
+      <em>Example:</em> <code>"\:hammer\: Tests"</code><br>
+    </td>
+  </tr>
+  <tr>
+    <td><code>branches</code></td>
+    <td>
+      The <a href="/docs/pipelines/branch-configuration#branch-pattern-examples">branch pattern</a> defining which branches will include this command step in their builds.<br>
+      <em>Example:</em> <code>"master stable/*"</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>env</code></td>
+    <td>
+      A list of <a href="/docs/builds/environment-variables">environment variables</a> for this step. <br>
+      <em>Example:</em> <code>FOO: "bar"</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>agents</code></td>
+    <td>
+      A list of <a href="/docs/agent/cli-meta-data">meta-data</a> key value pairs to <a href="docs/agent/cli-start#agent-targeting">target specific agents</a> for this step. <br>
+      <em>Example:</em> <code>npm: "true"</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>plugins</code></td>
+    <td>
+      A list of plugins that you require for this step. <br>
+      <em>Example:</em> <code>docker-compose</code><br>
+      <em>Example:</em> <code>harrietgl/myplugin</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>artifact_paths</code></td>
+    <td>
+      The path or paths where <a href="/docs/agent/cli-artifact">artifacts</a> from this step will be uploaded. <br>
+      <em>Example:</em> <code>"logs/**/*;coverage/**/*"</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>parallelism</code></td>
+    <td>
+      The number of jobs that will run this step <a href="/docs/builds/parallelizing-builds#job-parallelism">simultaneously</a>. <br>
+      <em>Example:</em> <code>1</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>concurrency</code></td>
+    <td>
+      The <a href="/docs/builds/controlling-concurrency#concurrency-limits">maximum number of jobs</a> that can be run inside this concurrency group at the same time as this job. <br>
+      <em>Example:</em> <code>1</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>concurrency_group</code></td>
+    <td>
+      A nickname for the concurrency value that you set with the <code>concurrency</code> attribute. This can also be the name of an existing <a href="/docs/builds/controlling-concurrency#concurrency-groups">concurrency group</a>. <br>
```

Comment:
There's a few quirks with this one that are probably worth documenting.

The concurrency value is organization wide, and I think people don't realise this because it's defined in a specific pipelines's pipeline.yml. So `deploy` in one pipeline will block a `deploy` in another pipeline.

An idea…

> The name of the group used to apply <a href="/docs/builds/controlling-concurrency#concurrency-groups">concurrency limits</a> when using the `concurrency` attribute. (something about it being org wide)